src/app/[category]/[product]/page.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import styles from './page.module.sass'
+
+import BackButton from '@/components/shared/BackButton'
+import ImageAndDescription from '@/components/shared/ImageAndDescription'
+import FeaturesAndBox from '@/components/PDP/FeaturesAndBox'
+import ImageGallery from '@/components/PDP/ImageGallery'
+import fetchGql from '@/lib/fetchGql'
+import { notFound } from 'next/navigation'
+import Recommendations from '@/components/PDP/Recommendations'
+import { queryProduct } from '@/lib/graphqlQueries'
+import { TypePDPContent } from '@/lib/types'
+
+const Page = async ({
+  params,
```

Comment:
What do you think of assigning the types to variables instead of inline?

Reply:
separated types into variables